packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+# 50.14.0 - Month DD, YYYY
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Remove channelService from core extension ([PR #2853](https://github.com/forcedotcom/salesforcedx-vscode/pull/2853))
+
+#### salesforcedx-vscode-soql
+
+- Do not emit message from custom editor to webview if soql has not changed ([PR #2818](https://github.com/forcedotcom/salesforcedx-vscode/pull/2818))
+
+## Fixed
+
+#### salesforcedx-utils-vscode
+
+- WorkspaceContext initialization across apex & replay-debugger packages ([PR #2868](https://github.com/forcedotcom/salesforcedx-vscode/pull/2868))
+
+#### salesforcedx-vscode-apex
+
+- WorkspaceContext initialization across apex & replay-debugger packages ([PR #2868](https://github.com/forcedotcom/salesforcedx-vscode/pull/2868))
+
+#### salesforcedx-vscode-apex-replay-debugger
+
+- WorkspaceContext initialization across apex & replay-debugger packages ([PR #2868](https://github.com/forcedotcom/salesforcedx-vscode/pull/2868))
```

Comment:
We might be able to consolidate these two into

```
#### salesforcedx-vscode-apex

- Fix issue running tests when session first starts ([PR #2868](https://github.com/forcedotcom/salesforcedx-vscode/pull/2868))
```

cc @AnanyaJha for better wording and confirmation